rs-soap-shop/src/components/forms/newFormComponents/ControlledInput.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ChangeEvent, useState } from 'react';
+import { IControlledInputContainer } from '../../../lib/interfaces';
+import { ValidFieldNames } from '../../../lib/types';
+import { HidePassword } from '../../../icons/hidePassword';
```

Comment:
Would please change naming for this icons : like HidePasswordIcon or smth. Now it looks like a function, not a component